### PR TITLE
AT_07.02_007 | Build History>Verify Sorting Builds

### PR DESCRIPTION
**Pull request by KseniyaParfianovich**
https://trello.com/c/f5DQdill/1433-at0702007-build-historyverify-sorting-builds

### CODE REVIEW

**Review by matvlesia — APPROVED** [review]

File: cypress/e2e/buildHistorySortBuilds.cy.js [hunk 1]
```diff
@@ -116,6 +116,23 @@ describe('Build History Sort builds', () => {
...
+    
+    it('AT_07.02_007 | Build History>Verify Sorting Builds', () => {
+        const buildsNumber = 3;
+        createBuildsOfNewProject(projects.newProject, buildsNumber); 
+
+        cy.get('a[href$="/builds"]').click();
+        cy.get('.sortheader').contains('Build').dblclick();
+        cy.get('table#projectStatus td:nth-child(2)').then(($els) => {
+        let actualStates = Cypress.$.makeArray($els).map(($el) => $el.innerText);
+        console.log(actualStates)
```
Comment (Rom-T) [inline cypress/e2e/buildHistorySortBuilds.cy.js:128]: Please delete `console.log`

Comment (KseniyaParfianovich) [reply cypress/e2e/buildHistorySortBuilds.cy.js]: Thank you. I've deleted it.

File: cypress/e2e/buildHistorySortBuilds.cy.js [hunk 2]
```diff
@@ -144,4 +144,3 @@ describe('Build History Sort builds', () => {
             expect(actualResult).to.deep.equal(expectedResult)
         })
     })
-});
```
Comment (Rom-T) [inline cypress/e2e/buildHistorySortBuilds.cy.js:130]: Why did you remove the closing brackets?

File: cypress/e2e/buildHistorySortBuilds.cy.js [hunk 3]
```diff
@@ -127,4 +143,3 @@ describe('Build History Sort builds', () => {
...
```
Comment (Rom-T) [inline cypress/e2e/buildHistorySortBuilds.cy.js:130]: Why did you remove the closing brackets? Please delete at 132 and leave at 146

Comment (KseniyaParfianovich) [reply cypress/e2e/buildHistorySortBuilds.cy.js:130]: I'm sorry. It happened when I solved a conflict. I didn't understand what brackets I need to remove. Thanks for your comment. I've changed it.

**Review by Rom-T — APPROVED** [review]
